api/graphite.go

```diff
@@ -847,6 +850,11 @@ func (s *Server) executePlan(ctx context.Context, orgId uint32, plan *expr.Plan)
 		resolveSeriesRequests[strippedreq] = append(resolveSeriesRequests[strippedreq], plan.Reqs[i])
 	}
 
+	// Output to see how useful this optimization might be
+	if len(resolveSeriesRequests) != len(plan.Reqs) {
+		log.Infof("executePlan: traceID=%s Making %d roundtrips to resolve %d reqs", traceID, len(resolveSeriesRequests), len(plan.Reqs))
```

Comment:
Same question with above, is it possible the traceID here is empty? Do we still want to log in that case?

Reply:
Yeah, for the same reason as above.